main/guides/getting-started/sell-concert-tickets-contract-explainer.md

```diff
@@ -43,20 +85,44 @@ const inventorySeat = ticketMint.mintGains(toMint);
 ```
 
 ## Trading Tickets
-Customers who wish to purchase event tickets first make an invitation to trade for tickets using `makeTradeInvitation`:
+
+Customers who wish to purchase event tickets first make an invitation to trade for tickets using `makeTradeInvitation`. Note that it calls `makeInvitation` function from [Zoe Contract Facet](https://docs.agoric.com/reference/zoe-api/zoe-contract-facet.html#zoe-contract-facet-zcf) to generate a **credible** invitation to a smart contract.
+
 ```js
 const makeTradeInvitation = () =>
   zcf.makeInvitation(tradeHandler, 'buy tickets', undefined, proposalShape);
 ```
 
-The `tradeHandler` function is called when a purchaser makes an offer:
+<details>
+<summary>
```

Comment:
We don't need this detail because `makeInvitation` API documentation has it.

Reply:
fixed